en/flight_controller/beaglebone_blue.md

```diff
@@ -28,40 +28,51 @@ The recommended way to build PX4 for *BeagleBone Blue* is to compile on a develo
 
 > **Tip** This approach is recommended over [native build](#native_builds) due to speed of deployment and ease of use.
 
+<span></span>
+> **Note** The PX4 build requires [librobotcontrol](http://strawsondesign.com/docs/librobotcontrol/) which is automatically included in the build (but it can be installed and tested independently if required).
+
+
 ### Beaglebone Blue WIFI Setup
 
 For easy access to your board, you can connect it to your home network via wifi.
 
-The steps are:
-```
-sudo -su
+The steps are (execute on the board):
+```sh
+sudo su
 connmanctl
 connmanctl>scan wifi
 connmanctl>services
-(at this point you should see your network appear along with other stuff, in my case it was "AR Crystal wifi_f45eab2f1ee1_6372797774616c_managed_psk")
+#(at this point you should see your network SSID appear.)
 connmanctl>agent on
-connmanctl>connect wifi_f45eab2f1ee1_6372797774616c_managed_psk
+connmanctl>connect <SSID>
 connmanctl>quit
 ```
 ### SSH root Login on Beaglebone
-With USB Serial connected 
-```
+
+Root login can be enabled on the board with:
+```sh
+sudo su
 echo "PermitRootLogin yes" >>  /etc/ssh/sshd_config && systemctl restart sshd
 ```
 
 ### Cross Compiler Setup
 
 1. First set up *rsync* (this is is used to transfer files from the development computer to the target board over a network - WiFi or Ethernet). 
-   For *rsync* over SSH with key authentication, follow steps here:
-   1. sudo su
-   1. ssh-keygen -t rsa -C root@beaglebone
-   1. ENTER //no passphrase
-   1. ENTER
-   1. ENTER
-   1. ssh-copy-id root@beaglebone
-   1. When promted if you trust - yes
-   1. password root
-   1. On the development computer, define the BeagleBone Blue board as `beaglebone` in **/etc/hosts**
+   For *rsync* over SSH with key authentication, follow the steps here (on the development machine):
+   1. Generate an SSH key if you have not previously done so: 
+      ```
+      ssh-keygen -t rsa
+      ```
+      1. ENTER //no passphrase
+      1. ENTER
+      1. ENTER
+   1. Define the BeagleBone Blue board as `beaglebone` in **/etc/hosts**:
+      ```
```

Comment:
@bkueng In your version there were a number of steps here - but I think this was intended to be part of this step.

Reply:
No, `Define the BeagleBone Blue board as `beaglebone` in **/etc/hosts**` is its own (optional) step. `ssh-copy-id root@beaglebone` Is required for password-less ssh access.

Reply:
@bkueng I'm not clear on the change - including whether the prompt to trust newting. Please confirm this is correct, or make the change yourself in source :-)

   1. Define the BeagleBone Blue board as `beaglebone` in **/etc/hosts**:
      ```
      ssh-copy-id root@beaglebone
      ```
   1. Alternatively you can use the beaglebone's IP for password-less SSH access:
      ```
      ssh-copy-id root@<IP>
      ```
      * When prompted if you trust: yes

Reply:
Like this (you'll be prompted when executing `ssh-copy-id`):

   1. Define the BeagleBone Blue board as `beaglebone` in **/etc/hosts** and copy the public SSH key to the board for password-less SSH access:
      ```
      ssh-copy-id root@beaglebone
      ```
   1. Alternatively you can use the beaglebone's IP directly:
      ```
      ssh-copy-id root@<IP>
      ```
   1. When prompted if you trust: yes
   1. Enter root password

Reply:
Thanks. Copied in https://github.com/PX4/px4_user_guide/commit/1f4471ec1328041560af8cb6e91bd2a030912a72